msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
```

Comment:
To be clear, these components (plus the simplified installation) were the reason for gitsetup.exe existed.